### ISSUE TITLE
Warning about obsolete entry `version` in docker-compose.yml

### ISSUE THREAD

**Issue by PeterCodar** [metabrainz/musicbrainz-docker#289]
Just imported the newest dump files and got this WARNING:

WARN[0000] /home/musicbrainz/musicbrainz-docker/docker-compose.yml: **`version` is obsolete** 

No need to worry, I just wanted to inform you.

Environment: 
Docker version 26.1.3, build 26.1.3-0ubuntu1~24.04.1
Description:	Ubuntu 24.04.1 LTS

**Comment by reosarevok** [comment]
Yeah, version should be removed while we're fully done updating to docker compose v2 :)
